### PR TITLE
formatting on rabbitmq credentials

### PR DESCRIPTION
**Pull request by xiujiao**

### CODE REVIEW

**Comment by xiujiao** [comment]
Will update documentation later

File: README.md [hunk 1]
```diff
@@ -167,22 +167,32 @@ applications:
...
+  "rmq_port": 5672,
+  "uri": "amqp://some-user:sekrit@10.x.x.x:5672",
+  "username": "some-user"
 }
-```
 
-- `rmq_port` - The TCP port that RabbitMQ's AMQP interface is
-  bound to and listening for inbound connections on.
+```
+- `host/hostname` - The IP address of the standalone RabbitMQ host.
```
Comment (jhunt) [inline README.md]: The `teletext` formatting made me initially think that `host/hostname` was the literal key name.  In reality, `host` and `hostname` are both interchangeable -- can we update the formatting to convey this?

File: README.md [hunk 2]
```diff
@@ -167,22 +167,32 @@ applications:
...
+    "host": "10.x.x.x",
+    "password": "sekrit",
+    "port": 5672,
+    "uri": "amqp://some-user:sekrit@10.x.x.x:5672",
+    "username": "some-user"
+   }
+  },
+  "rmq_port": 5672,
+  "uri": "amqp://some-user:sekrit@10.x.x.x:5672",
+  "username": "some-user"
```
Comment (jhunt) [inline README.md]: Can you match the existing formatting and line up the `:`?  It should make both the diff patch and the docs easier to read.

File: README.md [hunk 3]
```diff
@@ -192,21 +202,37 @@ applications:
   RabbitMQ.  It contains all of the other values, in a URL / DSN
   for easier integration with libraries / SDKs.
 
+The section in the protocols block is to also provide Pivatol RabbitMQ equivalent
```
Comment (jhunt) [inline README.md]: "Pivotal" is mispelled.

File: README.md [hunk 4]
```diff
@@ -192,21 +202,37 @@ applications:
...
+credentials formatting to applications. This is in order that applications 
+written to work with Pivotal RabbitMQ may work without changes to the applciations.
```
Comment (jhunt) [inline README.md]: Can you reword this?

File: README.md [hunk 5]
```diff
@@ -192,21 +202,37 @@ applications:
...
+    "uri": "amqp://some-user:sekrit10.x.x.x:5672",
+    "uris": ["amqp://some-user:sekrit@10.x.x.x:5672",
+             "amqp://some-user:sekrit@10.x.x.y:5672",
+             "amqp://some-user:sekrit@10.x.x.z:5672"]
+    "username": "some-user"
+   }
+  },
+  "rmq_port": 5672,
+  "uri": "amqp://some-user:sekrit10.x.x.x:5672",
+  "username": "some-user"
```
Comment (jhunt) [inline README.md]: As above, please match formatting.

File: README.md [hunk 6]
```diff
@@ -232,6 +258,11 @@ applications:
   accessing each cluster node individually.  Resilient
   applications should use this list to failover on node failure.
 
+
+The section in the protocols block is to also provide Pivatol RabbitMQ equivalent
```
Comment (jhunt) [inline README.md]: "Pivotal" is misspelled.

File: jobs/rabbitmq-blacksmith-plans/templates/plans/cluster/credentials.yml [hunk 7]
```diff
@@ -7,3 +7,15 @@ credentials:
...
+  protocols:
+    amqp:
+      hosts: (( grab jobs.node.ips ))
+      host: (( grab credentials.hosts[0] ))
+      password: (( grab meta.password ))
+      port: 5672
+      ssl: false
+      uris: (( grab credentials.uris ))
+      uri:  (( grab credentials.uris[0] ))
+      username: (( grab meta.username ))
```
Comment (jhunt) [inline jobs/rabbitmq-blacksmith-plans/templates/plans/cluster/credentials.yml]: Can you line up the opening `((` to match the existing style?

File: README.md (hunk 1 above)
Comment (xiujiao) [reply README.md]: fixed :)

File: README.md (hunk 2 above)
Comment (xiujiao) [reply README.md]: yes, done

File: README.md (hunk 3 above)
Comment (xiujiao) [reply README.md]: whoops, fixed

File: README.md (hunk 4 above)
Comment (xiujiao) [reply README.md]: I deleted second sentence

File: jobs/rabbitmq-blacksmith-plans/templates/plans/cluster/credentials.yml (hunk 7 above)
Comment (xiujiao) [reply jobs/rabbitmq-blacksmith-plans/templates/plans/cluster/credentials.yml]: done

**Comment by xiujiao** [comment]
@jhunt Could you please take a took when you get a chance to see if I addressed all your comments? After you are good with it, would like to merge and cut a final release.

**Pull request merged**